### PR TITLE
Add German Localization

### PR DESCRIPTION
**Pull request by Banane9**
Some of the keys could be removed too ... sadly the settings don't (yet) show descriptions for display items or groups either :/

### CODE REVIEW

File: MonkeyLoader.Resonite.Integration/Locale/de.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.MonkeyCount.Value": "{available, plural, =0{Keine Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Monkey} other{aktive Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.EarlyMonkeyCount.Value": "{available, plural, =0{Keine Early Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Early Monkey} other{aktive Early Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Name": "MonkeyLoader Einstellungen Öffnen",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Description": "Öffnet die Einstellungen für den Modloader selbst.",
+
+    "MonkeyLoader.GamePacks.Resonite.Mod.Metadata": "Metadaten",
+    "MonkeyLoader.GamePacks.Resonite.Mod.GroupDescription": "Zeigt eine Übersicht über den Mod.",
+    "MonkeyLoader.GamePacks.Resonite.Mod.OpenSettings": "Mod Einstellungen Öffnen",
+    "MonkeyLoader.GamePacks.Resonite.Mod.Description": "Beschreibung",
+    "MonkeyLoader.GamePacks.Resonite.Mod.Authors": "{count, plural, one{Author} other{Authoren}}",
```
Comment (ColinTimBarndt) [inline MonkeyLoader.Resonite.Integration/Locale/de.json:16]: [Autor ohne "h"](https://www.duden.de/suchen/dudenonline/Autor)

File: MonkeyLoader.Resonite.Integration/Locale/en.json
Comment (ColinTimBarndt) [inline MonkeyLoader.Resonite.Integration/Locale/en.json:1]: This is not a German localization, it should ideally be split off into another commit to avoid accidentally affecting this file when rolling back or cherrypicking commits. Was this accidentally staged and committed?

File: MonkeyLoader.Resonite.Integration/Locale/de.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "messages": {
+    "Settings.MonkeyLoader.Breadcrumb": "Einstellungen",
+
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.MonkeyCount.Value": "{available, plural, =0{Keine Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Monkey} other{aktive Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.MonkeyLoader.EarlyMonkeyCount.Value": "{available, plural, =0{Keine Early Monkeys} other{<b>{active}</b> {active, plural, one{aktiver Early Monkey} other{aktive Early Monkeys}} von <b>{available}</b> verfügbaren}}.",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Name": "MonkeyLoader Einstellungen Öffnen",
+    "MonkeyLoader.GamePacks.Resonite.OpenMonkeyLoader.Description": "Öffnet die Einstellungen für den Modloader selbst.",
+
+    "MonkeyLoader.GamePacks.Resonite.Mod.Metadata": "Metadaten",
+    "MonkeyLoader.GamePacks.Resonite.Mod.GroupDescription": "Zeigt eine Übersicht über den Mod.",
```
Comment (ColinTimBarndt) [inline MonkeyLoader.Resonite.Integration/Locale/de.json:13]: **die** Mod(ifikation)

File: MonkeyLoader.Resonite.Integration/Locale/de.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "MonkeyLoader.GamePacks.Resonite.WorldNotUserspace": "Diese Kategorie ist nur im Userspace verfügbar.",
+
+    "MonkeyLoader.GamePacks.Resonite.Mod.OpenConfigSections": "Verfügbare Einstellungen Öffnen",
+    "Settings.ConfigSections.Breadcrumb": "Einstellungen",
+
+    "MonkeyLoader.GamePacks.Resonite.Mod.OpenMonkeyToggles": "Verfügbare Monkeys Öffnen",
+
+    "MonkeyLoader.GamePacks.Resonite.Monkeys.Description": "Listet die Details aller verfügbaren Monkeys dieses Mods auf und erlaubt es sie zu (de)aktivieren.",
+    "MonkeyLoader.GamePacks.Resonite.Monkeys.Count.Name": "Anzahl",
+    "MonkeyLoader.GamePacks.Resonite.Monkeys.Count.Description": "Anzahl der Monkeys in diesem Mod.",
```
Comment (ColinTimBarndt) [inline MonkeyLoader.Resonite.Integration/Locale/de.json:31]: **dieser** Mod(ifikation)

**Review by ColinTimBarndt — CHANGES_REQUESTED** [review]
See my comments on minor grammatical errors, and I think the changes to the English localization shouldn't be in a PR for the German one.

File: MonkeyLoader.Resonite.Integration/Locale/de.json (hunk 2 above)
Comment (ColinTimBarndt) [reply MonkeyLoader.Resonite.Integration/Locale/de.json:13]: Also in line 13

File: MonkeyLoader.Resonite.Integration/Locale/en.json
Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Locale/en.json:1]: No, it was changed as part of this to make them a bit more similar lol

File: MonkeyLoader.Resonite.Integration/Locale/de.json (hunk 2 above)
Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Locale/de.json:13]: Die Meinungen sind gespalten 😂
https://www.duden.de/rechtschreibung/Mod

File: MonkeyLoader.Resonite.Integration/Locale/de.json (hunk 1 above)
Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Locale/de.json:16]: Oh, ja 😂

File: MonkeyLoader.Resonite.Integration/Locale/de.json (hunk 2 above)
Comment (ColinTimBarndt) [reply MonkeyLoader.Resonite.Integration/Locale/de.json]: Have you read the meaning of "Mod, der"?

> Angehöriger einer Gruppe männlicher Jugendlicher, die den Musikstil der 1960er-Jahre und als Kleidung Anzug und Krawatte bevorzugen

Translated:

> Member of a group of male teenagers who prefer the music style of the 1960s and dress in suits and ties

Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Locale/de.json]: Auf wiktionary steht auch für Mod (modifikation) `f, m` - aber ich hab's doch auch schon geändert xP

**Review by ColinTimBarndt — APPROVED** [review]
LGTM!! 😄
